.circleci/config.yml

```diff
@@ -27,11 +27,14 @@ jobs:
           command: |
             virtualenv venv
             source venv/bin/activate
-            pip install -r requirements.txt
-            flake8 . --exclude=venv,.venv --statistics
+            pip3 install flake8
+            flake8 . --count --exit-zero --max-complexity=10 --statistics
       - run:
           command: |
-            echo "test"
+            virtualenv venv
+            source venv/bin/activate
+            pip3 install pytest
+            pytest tests/
```

Comment:
Hey @PVinay737. We'll have several services in the repo which will have their own tests. 
For example, the directory structure would be
/service1
  /tests
/service2
  /tests

and so on. 
So I would recommend just use `pytest`

Reply:
It will automatically detect all tests directories and run those tests.

Reply:
Updated